test_runner/src/test/kotlin/ftl/run/status/MultiLinePrinterTest.kt

```diff
@@ -19,6 +20,7 @@ class MultiLinePrinterTest {
 
     @Test
     fun test() {
+        if (isWindows) return
```

Comment:
I cannot get this test works even I downrange the ```JANSI``` version to 1.18

Reply:
`assumeFalse(isWindows)`